x-pack/plugin/esql/qa/server/mixed-cluster/build.gradle

```diff
@@ -30,23 +30,23 @@ def supportedVersion = bwcVersion -> {
   return bwcVersion.onOrAfter(Version.fromString("8.11.0"));
 }
 
-// Versions on and after 8.13.3 will get a `version` parameter
-def versionUnsupported = bwcVersion -> {
-  return bwcVersion.before(Version.fromString("8.13.3"));
+// Versions on and after 8.14.0 will get a `version` parameter
+def versionRequired = bwcVersion -> {
```

Comment:
FWIW, I feel like this can (and likely should) be encoded in the tests and not in the build script. We should just be able to inject the version under test, and the tests figure out everything else.